src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -365,6 +365,15 @@ enum OpcodeClient : uint16
     CMSG_GUILD_BANK_REMAINING_WITHDRAW_MONEY_QUERY    = 0x3084,
     CMSG_GUILD_BANK_SET_TAB_TEXT                      = 0x3087,
     CMSG_GUILD_BANK_TEXT_QUERY                        = 0x3088,
+    CMSG_GUILD_BANK_TRANSFER_ITEM_TO_BANK             = 0x34BD,
+    CMSG_GUILD_BANK_TRANSFER_ITEM_TO_PLAYER           = 0x34BE,
+    CMSG_GUILD_BANK_MOVE_ITEM_IN_BANK                 = 0x34C1,
+    CMSG_GUILD_BANK_COMBINE_ITEM_STACK_WITH_BANK      = 0x34C2,
+    CMSG_GUILD_BANK_TRANSFER_ITEM_STACK_TO_BANK       = 0x34C3,
+    CMSG_GUILD_BANK_COMBINE_ITEM_STACK_WITH_PLAYER    = 0x34C4,
+    CMSG_GUILD_BANK_TRANSFER_ITEM_STACK_TO_PLAYER     = 0x34C5,
+    CMSG_GUILD_BANK_TRANSFER_ITEM_TO_PLAYER_AUTOSTORE = 0x34C6,
+    CMSG_GUILD_BANK_COMBINE_ITEM_STACK_IN_BANK        = 0x34C7,
```

Comment:
There seems to be missing an opcode here with value 0x34C8

Reply:
I could not trigger the opcode so I just left it out^^

Reply:
`CMSG_GUILD_BANK_MOVE_ITEMS_BANK_BANK_COUNT` is `0x34C8`

Reply:
And what is MOVE_ITEMS_BANK_BANK_COUNT for?

Reply:
![image](https://user-images.githubusercontent.com/291278/114843542-9a30ef00-9de2-11eb-8584-2aa982aae52e.png)
![image](https://user-images.githubusercontent.com/291278/114843571-a0bf6680-9de2-11eb-89f7-0465a6d6eb50.png)
![image](https://user-images.githubusercontent.com/291278/114843587-a4eb8400-9de2-11eb-956f-7844ddebbea6.png)

I saw this implementation floating around

Reply:
> And what is MOVE_ITEMS_BANK_BANK_COUNT for?

No idea, that's how I have it referenced in my IDB

Reply:
There should be 12 of these opcodes total, I can only see 9 in this PR

Reply:
> There should be 12 of these opcodes total, I can only see 9 in this PR

_Playing Gotta catch em' all Pokémon Theme Song_